### ISSUE TITLE
[CONTENT] Using NFTs in games

### ISSUE THREAD

**Issue by jessicaschilling** [protocol/nft-website#26]
This issue is for drafting content for the following:
- Title: Using NFTs in games
- Address: `/tutorial/using-nfts-in-games/`

**Contributions are welcome!** If you'd like to make the global NFT developer community better for everyone by creating a guide for this page, please leave a comment and someone from the core team will be in touch ASAP. 🙏

**Comment by michaeljohnbennett** [comment]
I'd like to get involved in this. I'm working on the consensys bootcamp and am working on a prototype using NFTs in a game mechanics.

**Comment by dchoi27** [comment]
@michaeljohnbennett awesome! would love to get you involved. Do you have something in mind for what could make a good tutorial for beginners?

**Comment by michaeljohnbennett** [comment]
I'm working on a project for the consensys blockchain bootcamp and I have this NFT game idea I'm developing.

I was thinking it could be a good base for a tutorial. Showing the general logic from minting packs, cards using probabilities.

also some simple on chain logic to do a game. The things I'm working on right now are VRF Oracles (maybe chainlink) but open to ideas on that side.

Sound like something worth working on/with?

**Comment by dchoi27** [comment]
Yeah that sounds awesome. As you make progress with your project, let us know and we can support your time to create a tutorial for the basics, if that sounds good! And maybe visit some of the more advanced stuff after?

**Comment by dchoi27** [comment]
Yeah I think a fork of the repo would be best to start adding the content! Could save some round trips if you wouldn't mind outlining the sections in this issue that you had in mind for the tutorial so we could align on flow. cc @yusefnapora 

Not sure how much code you have written, but if you have something up and working, we could support your work building a tutorial via a microgrant: https://github.com/filecoin-project/devgrants/blob/master/microgrants/microgrants.md cc @mishmosh

Side question - is your project using NFT.Storage? We think it's the easiest tool for the NFT School audience to get hands-on with development, so our goal is to standardize tutorial content using the NFT.Storage libraries.

**Comment by michaeljohnbennett** [comment]
I haven't seen that yet, I guess its some kind of IPFS storage container? I can use that for sure.

I'll fork the repo this weekend.

I have 3 smart contracts done and tested: 

- The store contract that allows you to buy packs
- The pack NFT which allows you to purchase packs
- The cards NFT which you get to play.

I don't have the pack opening random code yet done.

**Comment by dchoi27** [comment]
Yup, please check it out - it's a cool product!

Awesome, that sounds like plenty for a tutorial!

**Comment by CryptoMitch** [comment]
> I'd like to contribute to this, I'm currently working on two NFT games at the moment.
